docs/api/address/get-address-transactions-with-transfers.example.json

```diff
@@ -67,6 +67,22 @@
           "sender": "SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE",
           "recipient": "SP26066SDPP4NXKGCVYZQDR5GX2QPE8KADZ0YK2J7"
         }
+      ],
+      "ft_transfers": [
+        {
+          "amount": "10.53",
```

Comment:
Ah one last thing -- this will always be an integer, since we don't evaluate possible `decimals` defined for a given FT. It would probably be useful to note that in the schema description, something like "Note this balance does not factor in possible SIP-010 `decimals`".